packages/lab/src/components/Wizard/WizardActions/WizardActions.tsx

```diff
@@ -0,0 +1,194 @@
+import { useCallback, useContext, useEffect, useState } from "react";
+import { ClassNames } from "@emotion/react";
+import { clsx } from "clsx";
+import {
+  HvBaseProps,
+  HvButton,
+  HvDialogActions,
+  HvGrid,
+} from "@hitachivantara/uikit-react-core";
+import { Backwards, Forwards } from "@hitachivantara/uikit-react-icons";
+import wizardActionsClasses, {
+  HvWizardActionsClasses,
+} from "./wizardActionsClasses";
+import { styles } from "./WizardActions.styles";
+import { WizardContext } from "../WizardContext/WizardContext";
+
+export interface HvWizardActionsProps extends HvBaseProps {
+  /** Current tab to check if it's last page or first to disable previous button and swap between next and submit button. */
+  tab?: number;
+  /** Function to change the tab when pressing previous and next buttons. */
+  changeTab?: any;
+  /** Function to handle the cancel button. */
+  handleClose: (
+    event: React.SyntheticEvent,
+    reason?: "escapeKeyDown" | "backdropClick"
+  ) => void;
+  /** Function to handle the submit button. Also sends the current context state. */
+  handleSubmit: (context: any) => void;
+  /** An object containing all the labels for the wizard actions component. */
+  labels?: {
+    /** Cancel button label. */
+    cancel: string;
+    /** Skip button label. */
+    skip: string;
+    /** Previous button label. */
+    previous: string;
+    /** Next button label. */
+    next: string;
+    /** Submit button label. */
+    submit: string;
+  };
+  /** Whether the submit button is disabled. */
+  loading?: boolean;
+  /** Enables the skip button. */
+  skippable?: boolean;
+  /** A Jss Object used to override or extend the styles applied to the empty state Wizard. */
+  classes?: HvWizardActionsClasses;
+}
+
+export const HvWizardActions = ({
+  classes,
+  handleClose,
+  handleSubmit,
+  loading = false,
+  skippable = false,
+  labels = {
+    cancel: "Cancel",
+    next: "Next",
+    previous: "Previous",
+    skip: "Skip",
+    submit: "Submit",
+  },
+}: HvWizardActionsProps) => {
+  const { context, updateContext, tab, setTab } = useContext(WizardContext);
+  const [pages, setPages] = useState(0);
+  const [canSubmit, setCanSubmit] = useState(false);
+
+  useEffect(() => {
+    const contextEntries = Object.entries(context);
+    if (contextEntries.length) {
+      setPages(contextEntries.length);
+
+      const validWizard = Object.entries(context).every(
+        ([, value]) => value.valid
+      );
+      if (validWizard !== canSubmit) {
+        setCanSubmit(validWizard);
+      }
+    }
+  }, [context]);
+
+  const lastPage = pages - 1;
+  const isLastPage = tab >= lastPage;
+
+  const handleSkip = useCallback(() => {
+    const skippedContext = Object.entries(context).map(([, child]) => ({
+      ...child,
+      valid: child.valid !== false,
+    }));
+    updateContext(skippedContext);
```

Comment:
Check [this line](https://github.com/lumada-design/hv-uikit-react/pull/3297/files#diff-1ef21b5620f3174f5bd0e8891d621e118da7505aa9b995cb17b2d953b36c8e9cR52).

Reply:
You should do something like this:
```
const skippedContext = Object.entries(context).reduce(
   (acc, [key, child]) => ({
      ...acc,
       [key]: {
         ...child,
         valid: child.valid !== false,
       }
   }),
   {}
);
```